src/services/mongoose/users.ts

```diff
@@ -97,41 +102,38 @@ export const UsersRepository = (): IUsersRepository => {
   }
 }
 
-const userFromRaw = (result: UserType): User => {
-  return {
-    id: result.id as UserId,
-    username: result.username as Username,
-    walletPublicId: result.walletPublicId as WalletPublicId,
-    phone: result.phone as PhoneNumber,
-    language: result.language as UserLanguage,
-    twoFA: result.twoFA as TwoFAForUser,
-    contacts: result.contacts.reduce(
-      (res: UserContact[], contact: ContactObjectForUser): UserContact[] => {
-        if (contact.id) {
-          res.push({
-            username: contact.id as Username,
-            alias: (contact.name || contact.id) as ContactAlias,
-            transactionsCount: contact.transactionsCount,
-          })
-        }
-        return res
-      },
-      [],
-    ),
-    quizQuestions:
-      result.earn?.map(
-        (questionId: string): UserQuizQuestion => ({
-          question: {
-            id: questionId as QuizQuestionId,
-            earnAmount: toSats(onboardingEarn[questionId]),
-          },
-          completed: true,
-        }),
-      ) || [],
-    defaultAccountId: result.id as AccountId,
-    deviceTokens: (result.deviceToken || []) as DeviceToken[],
-    lastConnection: result.lastConnection,
-    lastIPs: (result.lastIPs || []) as IPType[],
-    createdAt: new Date(result.created_at),
-  }
-}
+const userFromRaw = (result: UserType): User => ({
+  id: result.id as UserId,
+  status: result.status as AccountStatus,
```

Comment:
Why is status added as part of this PR?

Reply:
I seen as missing, but I guess that might be intentional? it's currently only used from the admin panel AFAIR

Reply:
Aha admin panel is still using the old user type (coming from mongoose) not this one. No need to add it now.

Reply:
The admin panel user could and probably should be its own type with its own repository - as it serves different use cases than the user we are returning here.